content/docs/contributing/contributing-flow.md

```diff
@@ -117,7 +117,7 @@ If this PR introduces a breaking change, the release note block must start with
 ### Cherry Picking
 
 If the pull request contains a critical bug fix then this should be cherry picked in to the current stable cert-manager branch 
-and [released as a patch release](../installation/supported-releases.md#support-policy).
+and [released as a patch release](../releases#support-policy).
```

Comment:
This and all internal links should link to files so that the link checker can report broken links if the target file is moved again in the future.